eval/Containerfile

```diff
@@ -4,15 +4,16 @@ RUN dnf install -y python3.11 python3.11-devel git python3-pip make automake gcc
     && dnf clean all \
     && rm -rf /var/cache/*dnf*
 
-# TODO update to instructlab/eval after https://github.com/instructlab/eval/pull/128 or equivalent merges
+ENV LD_LIBRARY_PATH="$LD_LIBRARY_PATH:/usr/local/lib/python3.11/site-packages/nvidia/cudnn/lib"
+
 RUN python3.11 -m ensurepip \
-    && python3.11 -m pip install --no-cache-dir git+https://github.com/instructlab/eval@main \
+    && python3.11 -m pip install packaging wheel setuptools-scm torch psutil \
+    && python3.11 -m pip install flash_attn --no-build-isolation \
+    && python3.11 -m pip install vllm@git+https://github.com/opendatahub-io/vllm@2024.08.01 \
```

Comment:
How about setting a variable for both vllm and eval so next time we update, we just change the value instead of editing the installation lines (which is error prone).

Something like:

```
ENV VLLM_VERSION=2024.08.01
EVAL_VERSION=v0.2.1
```

Then in the install line do:

```
python3.11 -m pip install --no-cache-dir git+https://github.com/instructlab/eval@${EVAL_VERSION}
```

Reply:
I'll do that, but right now the vllm install is failing - i can't get it to build

Reply:
updated, using the ARG values fro EVAL_VERSION, VLLM_VERSION